test/test_transforms.py

```diff
@@ -28,7 +28,227 @@
+            assert t.interpolation==transforms.InterpolationMode.BILINEAR
+
+        with pytest.warns(UserWarning, r"Argument fillcolor is deprecated and will be removed"):
+            t = transforms.RandomAffine(10, fillcolor=10)
+            assert t.fill==10
+
+        # assert changed type warning
+        with pytest.warns(UserWarning, r"Argument interpolation should be of type InterpolationMode"):
+            t = transforms.RandomAffine(10, interpolation=2)
+            assert t.interpolation==transforms.InterpolationMode.BILINEAR)
+    
+
+        torch.testing.assert_close(output, expected_output)
+
+@pytest.mark.skipif(accimage is None, 'accimage not available')
+    def test_accimage_to_tensor():
+        trans = transforms.ToTensor()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+@pytest.mark.skipif(accimage is None, 'accimage not available')
+    def test_accimage_resize():
+        trans = transforms.Compose([
+            transforms.Resize(256, interpolation=Image.LINEAR),
+            transforms.ToTensor(),
+        ])
+
+        # Checking if Compose, Resize and ToTensor can be printed as string
+        trans.__repr__()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        assert expected_output.size()==output.size())
+        assertLess(np.abs((expected_output - output).mean()), 1e-3)
+        assertLess((expected_output - output).var(), 1e-5)
+        # note the high absolute tolerance
+        assert(np.allclose(output.numpy(), expected_output.numpy(), atol=5e-2))
+
+@pytest.mark.skipif(accimage is None, 'accimage not available')
+    def test_accimage_pil_to_tensor():
+        trans = transforms.PILToTensor()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        assert expected_output.size()==output.size()
+        torch.testing.assert_close(output, expected_output, check_stride=False)
+
+@pytest.mark.skipif(accimage is None, 'accimage not available')
+    def test_accimage_crop():
+        trans = transforms.Compose([
+            transforms.CenterCrop(256),
+            transforms.ToTensor(),
+        ])
+
+        # Checking if Compose, CenterCrop and ToTensor can be printed as string
+        trans.__repr__()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        assert expected_output.size()==output.size()
+        torch.testing.assert_close(output, expected_output)
+		
+@pytest.mark.parametrize('a', range(-90,90,36))
+@pytest.mark.parametrize('t1',range(-10, 10, 5))
+@pytest.mark.parametrize('s',[0.77, 1.0, 1.27])
+@pytest.mark.parametrize('s1',range(-15, 15, 5))	
+@pytest.mark.parametrize(pt,[(16, 16), (20, 16), (20, 20)])
```

Comment:
this should be `'pt'` with quotes and you'll need a space after the comma ;) 

(here and in other places)